Source/AhriAndBear/Public/Component/AABSurvivalComponent.h

```diff
@@ -68,4 +68,5 @@ class AHRIANDBEAR_API UAABSurvivalComponent : public UActorComponent
 	void TickStat(float& currentValue, int maxValue, float delta);
 
 	FORCEINLINE FSurvivalData GetSurvivalData() const { return FSurvivalData{ CurrentHealth, CurrentFullness, CurrentThirst, CurrentWarmth }; }
```

Comment:
Also, this interface is for the EventTrigger's need, please rename it to your function.